src/templates/customization/custom-html.html

```diff
@@ -0,0 +1,109 @@
+<%
```

Comment:
could we include an external JavaScript file produced from typescript instead of writing JS inside HTML page ?